### PR TITLE
Added article for border-radius

### PR DESCRIPTION
**Pull request by chiragbhansali**

### CODE REVIEW

File: src/pages/css/css3-borders-rounded-corners/index.md [hunk 1]
```diff
@@ -3,13 +3,24 @@ title: CSS3 Borders Rounded Corners
 ---
 ## CSS3 Borders Rounded Corners
```
Comment (HKuz) [inline src/pages/css/css3-borders-rounded-corners/index.md]: Let's change the title to "CSS3 Border Radius Property"

File: src/pages/css/css3-borders-rounded-corners/index.md [hunk 2]
```diff
@@ -3,13 +3,24 @@ title: CSS3 Borders Rounded Corners
...
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/css3-borders-rounded-corners/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+With CSS3, you can give any element "rounded corners", by using the border-radius property. The value can be in any valid CSS unit.
```
Comment (HKuz) [inline src/pages/css/css3-borders-rounded-corners/index.md]: Add syntax highlighting so border-radius property becomes `border-radius` property

File: src/pages/css/css3-borders-rounded-corners/index.md [hunk 3]
```diff
@@ -3,13 +3,24 @@ title: CSS3 Borders Rounded Corners
...
+        border-radius: 20px;
+    }
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+    .circle {
+        border-radius: 50%;
+    }
+```
+
+**Note:** The border-radius property is actually a shorthand property for the border-top-left-radius, border-top-right-radius, border-bottom-right-radius and border-bottom-left-radius properties.
```
Comment (HKuz) [inline src/pages/css/css3-borders-rounded-corners/index.md]: Add syntax highlighting around the property names, so it's "... `border-top-left-radius`, `border-top-right-radius`, `border-bottom-right-radius`, and `border-bottom-left-radius` properties."

File: src/pages/css/css3-borders-rounded-corners/index.md [hunk 4]
```diff
@@ -3,13 +3,24 @@ title: CSS3 Borders Rounded Corners
...
+        border-radius: 50%;
+    }
+```
+
+**Note:** The border-radius property is actually a shorthand property for the border-top-left-radius, border-top-right-radius, border-bottom-right-radius and border-bottom-left-radius properties. 
 
 #### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
 
+Docs: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius)
```
Comment (HKuz) [inline src/pages/css/css3-borders-rounded-corners/index.md]: Change the "More Information" to a level-3 heading (`###` vs. `####`), and replace "Docs:" with "MDN Documentation:"

File: src/pages/css/css3-borders-rounded-corners/index.md [hunk 5]
```diff
@@ -3,13 +3,24 @@ title: CSS3 Borders Rounded Corners
 ---
```
Comment (HKuz) [inline src/pages/css/css3-borders-rounded-corners/index.md]: Change the title field to "CSS3 Border Radius Property"

File: src/pages/css/css3-borders-rounded-corners/index.md [hunk 6]
```diff
@@ -1,15 +1,26 @@
...
+        border-radius: 20px;
+    }
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+    .circle {
+        border-radius: 50%;
+    }
+```
+
+**Note:** The border-radius property is actually a shorthand property for the ```border-top-left-radius```, ```border-top-right-radius```, ```border-bottom-right-radius``` and ```border-bottom-left-radius``` properties.
```
Comment (texas2010) [inline src/pages/css/css3-borders-rounded-corners/index.md]: need to fix this. not three backticks.  Please use single backtick. thanks

**Comment by texas2010** [comment]
I fix some.

**Comment by texas2010** [comment]
it seems we have to use local for move file to another folder since you asked. after we merge it. I guess. then make PR for that.
> it should be moved into src/pages/css/properties

**Comment by HKuz** [comment]
@texas2010 - I tried to move the file from my interface, but I'm only seeing that it lets me change the file name, not the path. We can address the duplicates in a separate PR 👍. I just tweaked some of the formatting and I'm going to merge this. Thanks for another article @chiragbhansali ⚪️ 🔴 ⚫️

**Pull request merged**